cpataltmetrik/webdriveriowithtypescript
# Framework Feature: TARA Report Portal Integration with Groot

### Ticket Number / Feature:
- TARA Report Portal Integration with Groot

### Description:
- This feature is going to report the test execution on TARA portal.

### PR Check List:


### Test Execution Proof:
- Please see all the results on http://54.219.33.119:4000/ui/#hello/launches/all 

### Lint Result Proof:
- As of now, Lint implementation PR is created hence skipping for now.

### How to Execute Test Case:

```sh
1. Go to wdio.local.conf.ts file and add the file path under specs which we wanted to execute.
2. Run the command npm run test-dev for local exeution.
3. Check the execution results on TARA Portal by visiting the dashboard.
```
